### PR TITLE
Create article: azs-how-create-vm-from-snapshot-ps

### PR DESCRIPTION
**Pull request by wturner123**
Created article as per JIRA ticket AS-1477.

### CODE REVIEW

File: articles/azure/azs-how-create-vm-from-snapshot-ps.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+    $VirtualMachine.OSProfile.WindowsConfiguration.ProvisionVMAgent = $true
+}
+
+# Add Network Interface Card
+$VirtualMachine = Add-AzureRmVMNetworkInterface -Id $NetworkInterface.Id -VM $VirtualMachine
+
+# Applies the OS disk properties
+$VirtualMachine = Set-AzureRmVMOSDisk -VM $VirtualMachine -ManagedDiskId $NewDisk.Id -StorageAccountType "StandardLRS" -CreateOption "Attach" <output form="vmtype" name="result3" style="display: inline;">-Linux</output>
+
+# Enable boot diagnostics.
```
Comment (niall-lefley) [inline articles/azure/azs-how-create-vm-from-snapshot-ps.md:182]:
```suggestion
# Enable boot diagnostics
```

File: articles/azure/azs-how-create-vm-from-snapshot-ps.md [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+$VirtualMachine = Set-AzureRmVMBootDiagnostics -VM $VirtualMachine -Enable -StorageAccountName $SAName -ResourceGroupName $NewRGName
+
+# Create the virtual machine.
```
Comment (niall-lefley) [inline articles/azure/azs-how-create-vm-from-snapshot-ps.md:185]:
```suggestion
# Create the virtual machine
```

File: articles/azure/azs-how-create-vm-from-snapshot-ps.md [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+toc_title: Create a VM from a disk snapshot - PowerShell
+toc_fullpath: Users/How To/Create a VM from a Disk Snapshot/azs-how-create-vm-from-snapshot-ps.md
+toc_mdlink: azs-how-create-vm-from-snapshot-ps.md
+---
+
+# How to create a virtual machine from a disk snapshot using PowerShell
+
+## Overview
+
+This article shows you how to use PowerShell to take a snapshot of a managed disk that is attached to a virtual machine, then use the snapshot to create a new managed disk and attach it to a new virtual machine from the new disk.
```
Comment (suehighmoor) [inline articles/azure/azs-how-create-vm-from-snapshot-ps.md:23]: Include the same text here from the Portal version to describe why customers might want to do this?

Comment (wturner123) [reply articles/azure/azs-how-create-vm-from-snapshot-ps.md]: Updated in latest commit.

**Review by cblackuk — CHANGES_REQUESTED** [review]
It has been left (my fault) for so long it needs to be re-written and re-tested for Az modules.

**Comment by wturner123** [comment]
Updated to Az modules and tested. Some additional changes made to improve the article.

**Review by cblackuk — APPROVED** [review]
LGTM

**Pull request merged**